lms/djangoapps/course_home_api/progress/v1/tests/test_views.py

```diff
@@ -35,6 +35,8 @@ def test_get_authenticated_enrolled_user(self, enrollment_mode):
         self.assertIsNotNone(response.data['courseware_summary'])
         for chapter in response.data['courseware_summary']:
             self.assertIsNotNone(chapter)
+        self.assertIsNotNone(response.data['studio_url'])
+        self.assertTrue(str(self.course.id) in response.data['studio_url'])
```

Comment:
`self.assertIn(str(self.course.id, response.data['studio_url'])` and if you can, you could also maybe throw in the `settings/grading` since you know to expect those too.